src/jtagmkII.c

```diff
@@ -3601,6 +3601,37 @@ static int jtagmkII_flash_clear_pagebuffer32(const PROGRAMMER *pgm) {
     return -1;
 }
 
+/*** Periodic calls in terminal mode to keep `JTAGMKII_UPDI` enabled. ***/
+/*
```

Comment:
This 5-line comment basically says this function is only useful for `jtagmkii_updi` programmers. Better to remove the comment and *rename* the function to `jtagmkII_updi_term_keep_alive()`. This way the project generates self-documenting code.